http-server/src/main/java/io/airlift/http/server/HttpServerBinder.java

```diff
@@ -51,6 +53,18 @@ public HttpServerBinder enableLegacyUriCompliance()
         return this;
     }
 
+    public HttpServerBinder bindErrorHandler(ErrorHandler errorHandler)
```

Comment:
We don't want to expose Jetty specific classes. Instead, we should have on own, simpler interface here.

Reply:
What would the interface look like @electrum ?